libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/CoiSubjourneyTypes.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+import java.util.List;
+
+@Getter
+@ExcludeFromGeneratedCoverageReport
+public enum CoiSubjourneyTypes {
+    // Making changes to this enum? Let the data team know. Changes here will cause changes to the
+    // IPV_SUBJOURNEY_START audit event, which they consume
```

Comment:
Is this comment true? I think only the values in `IpvJourneyTypes` will affect that audit event?

Reply:
We will be using this for the new TXMA audit events for this flow, so we can probably remove it for now until then